src/App.js

```diff
@@ -10,7 +10,7 @@ import { useParams } from "react-router-dom";
 import SingleReview from "./pages/SingleReview";
 
 function App() {
-  const [username, setUser] = useState("mallionaire");
+  const [username, setUser] = useState("grumpy19");
```

Comment:
This is a global user state, so this is a perfect use case for React Context (Tuesday Lecture). 
Alternatively, you could hardcode this closer to the place where it's used to clean up your props.